CardScan.podspec

```diff
@@ -9,7 +9,8 @@ CardScan is a library for scanning credit cards.
   s.homepage         = 'https://cardscan.io'
   s.license          = { :type => 'BSD', :file => 'LICENSE' }
   s.author           = { 'Sam King' => 'kingst@gmail.com' }
-  s.source           = { :git => 'https://github.com/Ibotta/cardscan-ios.git', :tag => s.version.to_s }
+  s.source           = { :git => 'https://github.com/getbouncer/cardscan-ios.git', :tag => s.version.to_s } 
```

Comment:
Whoops. My fault! I should have excluded that when comparing against our fork. So sorry.